src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -117,132 +117,30 @@
 # import boto sdb connection for autoscaling
 import boto.sdb
 
+import textwrap
```

Comment:
What type of sorted order? I'm agnostic on this so I'm happy to do whatever you want.